db/migrate/20240903184807_migrate_tag_added_filter_to_all_providers.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
```

Comment:
We can't have any rails class in a migration, it should be all sql, if we change these class, migrations are borked

Reply:
This ^. You'll have to inline whatever `DiscourseChatIntegration::Provider.constants` resolves to.